pkgs/by-name/ma/mangareader/package.nix

```diff
@@ -1,54 +1,50 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, wrapQtAppsHook
-, extra-cmake-modules
 , cmake
-, kio
-, ki18n
-, kxmlgui
-, kconfig
-, karchive
-, kcoreaddons
-, kconfigwidgets
+, qt6
+, kdePackages
 , nix-update-script
 }:
 
 stdenv.mkDerivation rec {
   pname = "mangareader";
-  version = "2.1.0";
+  version = "2.2.1";
 
   src = fetchFromGitHub {
     owner = "g-fb";
-    repo = pname;
+    repo = "mangareader";
     rev = version;
-    hash = "sha256-YZZcp+HS/P/GxWYyOpO35nByJSzv4HahzzrZSVRcCRs=";
+    hash = "sha256-XX0VaXVYmAs5vmgwslflKIYx1peITp4VmReLkv1nV3I=";
   };
 
   nativeBuildInputs = [
     cmake
-    extra-cmake-modules
-    wrapQtAppsHook
+    kdePackages.extra-cmake-modules
+    qt6.wrapQtAppsHook
   ];
 
   buildInputs = [
+    qt6.qtbase
```

Comment:
You don't have to mix the two, there's `kdePackages.wrapQtAppsHook` and `kdePackages.qtbase`